packages/opam-monorepo/opam-monorepo.0.1.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis: "Assemble and manage fully vendored Dune repositories"
+description: """
+The opam monorepo plugin provides a convenient interface to bridge the
+opam package manager with having a local copy of all the source
+code required to build a project using the dune build tool."""
+maintainer: ["anil@recoil.org"]
+authors: [
+  "Anil Madhavapeddy" "Nathan Rebours" "Lucas Pluvinage" "Jules Aguillon"
+]
+license: "ISC"
+homepage: "https://github.com/ocamllabs/opam-monorepo"
+bug-reports: "https://github.com/ocamllabs/opam-monorepo/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.08.0" & < "4.12"}
```

Comment:
Is the `< 4.12` bound just to make the CI happy (avoid the dune incompatibility) or there are deeper reasons?

Reply:
Shouldn't that constraint be in the dune packages itself then?

Reply:
I have not followed the work toward 4.12, but my guess is that this will happen when the new dune is released to avoid a full recompilation of all opam switches in the meantime. I think @kit-ty-kate knows better

Reply:
There are deeper reasons as we vendor some of our dependencies in versions that appear to be incompatible with 4.12!

Reply:
I'll patch these up and release a 0.1.1 with 4.12 compat once dune is released!

Reply:
ah yes indeed. `lwt`, `stdlib-shims` and partially `dune` is incompatible at the moment.